packages/react-async/src/Provider.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {AsyncManager, Manager} from './manager';
+import {Prefetcher} from './Prefetcher';
+import {AsyncContext} from './context';
+
+interface Props {
+  children?: React.ReactNode;
+  manager?: Manager;
+}
+
+export class AsyncProvider extends React.PureComponent<Props> {
```

Comment:
Same here, maybe use `AsyncComponentProvider` to let readers know it's related to all `createAsyncComponent` that will be introduced app-wide.

Reply:
Same answer as above, it is not just for async components.